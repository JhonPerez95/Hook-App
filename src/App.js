import React from 'react';

// import FormCustomHook from './components/useEffect/FormCustomHook';
import MultipleCustomHooks from './components/exampless/MultipleCustomHooks';
// import SimpleForm from './components/useEffect/SimpleForm';
// import CounterApp from './components/useState/CounterApp';
// import CounterCustomHook from './components/useState/CounterCustomHook';

function App() {
  return (
    <>
      <MultipleCustomHooks />
    </>
  );
}

export default App;
