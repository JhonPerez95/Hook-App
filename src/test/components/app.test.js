import React from 'react';
import { shallow } from 'enzyme';
import App from '../../App';

describe('Prueba del componente Padre', () => {
  test('Renderizacion correcta del componente', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
