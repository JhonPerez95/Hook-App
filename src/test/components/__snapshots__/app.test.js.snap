// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba del componente Padre Renderizacion correcta del componente 1`] = `
<div>
  <MainApp />
</div>
`;
